Packs/MicrosoftGraphListener/Integrations/MicrosoftGraphListener/MicrosoftGraphListener_description.md

```diff
@@ -1,2 +1,24 @@
  To allow us access to Microsoft Graph Mail, you need to approve our app, by clicking on the following [link](https://oproxy.demisto.ninja/ms-graph-mail-listener).
  After authorizing the Demisto app, you will receive an ID, Refresh Token, and Key, which needs to be added to the integration instance configuration's corresponding fields.
+
+## Authorize Cortex XSOAR for Microsoft Graph Mail (self-deployed configuration)
+
+Follow these steps for a self-deployed configuration.
+
+1. To use a self-configured Azure application, you need to add a new MS Graph App Registration in the Azure Portal. To add the registration, refer to the following [Microsoft article](https://docs.microsoft.com/en-us/azure/active-directory/develop/quickstart-register-app).
+2. Make sure the following permissions are granted for the app registration:
+   -  API/Permission name `openid` of type `Delegated`
+   -  API/Permission name `profile` of type `Delegated`
+   -  API/Permission name `Mail.ReadWrite` of type `Delegated`
+   -  API/Permission name `Mail.Send` of type `Delegated`
+   -  API/Permission name `Mail.Read` of type `Delegated`
```

Comment:
did you make sure that these are the minimal permission set?

Reply:
I think that these are the permissions we have to include. 
@Itay4  am I right?

Reply:
these shouldnt be in here, but in the integration doc
also we already have this documented in https://xsoar.pan.dev/docs/reference/articles/microsoft-integrations---authentication so you should just refer to that article
see how its done in other microsoft integrations

Reply:
Ok, copied the description from MicrosoftGraphMail